counterpoll/main.py

```diff
@@ -203,9 +203,15 @@ def watermark():
     """ Watermark counter commands """
 
 @watermark.command()
-@click.argument('poll_interval', type=click.IntRange(1000, 30000))
+@click.argument('poll_interval', type=click.IntRange(1000, 60000))
 def interval(poll_interval):
-    """ Set watermark counter query interval for both queue and PG watermarks """
+    """ 
+    Set watermark counter query interval for both queue and PG watermarks 
```

Comment:
I would suggest to remove this verbose content regarding the interval change since this will show up in the command usage. We can capture this in the PR description